packs/salt/actions/lib/base.py

```diff
@@ -1,9 +1,23 @@
+# pylint: disable=no-member
+
 from st2actions.runners.pythonrunner import Action
 from requests import Request
+import logging
```

Comment:
The benefit of logger from st2common are a few extras so that is preferred in stackstorm context over std_lib logging. 

On the other hand the nicer way for an action is to go with the logger setup by the runner so `self.logger` in the Action like
https://github.com/StackStorm/st2/blob/master/contrib/packs/actions/pack_mgmt/download.py#L132.

Not doing so would mean that log lines may not end up in the action output since a console handler would not be provided by default. Of course passing a logger around everywhere can be very irritating and take away from code clarity.